src/universal-dapp-ui.js

```diff
@@ -31,31 +34,34 @@ UniversalDAppUI.prototype.renderInstance = function (contract, address, contract
 // basically this has to be called for the "atAddress" (line 393) and when a contract creation succeed
 // this returns a DOM element
 UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address, contractName) {
+  // VARIABLES
   var self = this
 
-  function remove () { instance.remove() }
-
   address = (address.slice(0, 2) === '0x' ? '' : '0x') + address.toString('hex')
-  var instance = yo`<div class="instance ${css.instance}" id="instance${address}"></div>`
+  var instance = yo`<div class="instance ${css.instance} ${css.hidesub}" id="instance${address}"></div>`
   var context = self.udapp.context()
-
   var shortAddress = helper.shortenAddress(address)
-  var title = yo`<div class="${css.title}" onclick=${toggleClass}>
-    <div class="${css.titleText}"> ${contractName} at ${shortAddress} (${context}) </div>
-    ${copyToClipboard(() => address)}
-  </div>`
+  var title = yo`
+    <div class="${css.title}" onclick=${toggleClass}>
+      <div class="${css.titleText}"> ${contractName} at ${shortAddress} (${context}) </div>
+      ${copyToClipboard(() => address)}
+    </div>`
 
+  instance.appendChild(title)
   if (self.udapp.removable_instances) {
     var close = yo`<div class="${css.udappClose}" onclick=${remove}><i class="${css.closeIcon} fa fa-close" aria-hidden="true"></i></div>`
-    instance.append(close)
+    title.append(close)
```

Comment:
A fix has been pushed on `master` , that should be `title.appendChild(close)`

Reply:
Thanks, fixed now!